doc/api/n-api.md

```diff
@@ -172,7 +172,7 @@ Function pointer type for user-provided native functions which are to be
 exposed to JavaScript via N-API. Callback functions should satisfy the
 following signature:
 ```C
-typedef void (*napi_callback)(napi_env, napi_callback_info);
+typedef napi_value (*napi_callback)(napi_env, napi_callback_info);
```

Comment:
This was already fixed in https://github.com/nodejs/node/pull/13570. You'll need to rebase.